### PR TITLE
feat: Optimize the current address highlighting style

### PR DESCRIPTION
**Pull request by zhb666**
### issues：https://github.com/Magickbase/ckb-explorer-public-issues/issues/317
Optimize the highlighting of the current address and enable users to quickly understand their corresponding address, while also adding a corresponding icon for visual indication.

image

### CODE REVIEW

File: src/components/TransactionItem/TransactionIncome/styled.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const TransactionCapacityValuePanel = styled.div`
   display: flex;
   justify-content: flex-end;
   align-items: center;
-  color: ${(props: { increased: boolean; theme: any }) => (props.increased ? props.theme.primary : '#FF6347')};
+  color: ${(props: { increased: boolean; theme: any }) => (props.increased ? '#333' : '#FF6347')};
```
Comment (Keith-CY) [inline src/components/TransactionItem/TransactionIncome/styled.tsx:23]: I didn't see the change of color in the design draft.
But I do remember a blame that `capacity` in `primary color` looks like clickable. Is this change for that blame? If yes, please add a reference.

**Review by Keith-CY — APPROVED** [review]

**Comment by FrederLu** [comment]
image
1、Compared with the `home` icon at the address in the screenshot, is the amount at the lower end?

---
image

2、After the `home` icon is added to the amount, the overall display on the mobile is no longer centered

**Comment by zhb666** [comment]
image

image

**Comment by Keith-CY** [comment]
Please have a review @WhiteMinds @IronLu233

File: src/components/TransactionItem/TransactionItemCell/styled.tsx [hunk 2]
```diff
@@ -52,21 +52,24 @@ export const TransactionCellCapacityPanel = styled.div`
   max-height: 40px;
   align-items: center;
 
+  .transaction__cell__without__icon {
```
Comment (IronLu233) [inline src/components/TransactionItem/TransactionItemCell/styled.tsx:55]: This class name seems very odd. in convention, the spreator is `-`, or `_`(BEM)

**Review by IronLu233 — APPROVED** [review]

**Comment by Keith-CY** [comment]
2 Approvals from developers, please have a review @FrederLu @littleLip520 @silySuper

**Comment by FrederLu** [comment]
On the mobile terminal, except `CKB`, other tokens (including
`Unknown Token #62bc`) the `current address` icon spacing is too small.

https://ckb-explorer-h11tq8r5i-magickbase.vercel.app/address/ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqvwknt4s90zypaa9xstxeg4y0j6fr506dqvz7cxm
image

https://ckb-explorer-h11tq8r5i-magickbase.vercel.app/address/ckt1qpuljza4azfdsrwjzdpea6442yfqadqhv7yzfu5zknlmtusm45hpuqgpu8g74pvw966l9kczvhjjts288hnk8ls3qqgf5p53
image

**Comment by zhb666** [comment]
The issue has been resolved. Could you please verify it again? Thank you. @FrederLu 
image

**Review by FrederLu — APPROVED** [review]

**Pull request merged**